packages/botonic-core/src/plugins.ts

```diff
@@ -32,11 +39,12 @@ export async function runPlugins(
   dataProvider?: DataProvider
 ): Promise<void> {
   for (const key in plugins) {
-    const p = await plugins[key]
+    const p: Plugin = await plugins[key]
```

Comment:
It would be nice to solve the `//TODO` we have in the line 32 of this same file and create a proper type for the `plugins` argument to avoid adding this `Plugin` type here.
Not to do it now, but in a near future :wink:

Reply:
added 👍